packages/core/src/BreadCrumb/BreadCrumb.js

```diff
@@ -3,10 +3,9 @@ import PropTypes from "prop-types";
 import clsx from "clsx";
 import { withStyles } from "@mui/styles";
 import { DropRightXS } from "@hitachivantara/uikit-react-icons";
-import startCase from "lodash/startCase";
 import isNil from "lodash/isNil";
 import { HvLink, HvTypography, HvOverflowTooltip } from "..";
-import { pathWithSubMenu, removeExtension } from "./utils";
+import { pathWithSubMenu, removeExtension, startCase } from "./utils";
```

Comment:
I'm unsure if the original intent was to also remove `_.-` or just to capitalise, but `startCase` is also removing any other (it seems like) non-ASCII character, making it harder or impossible to correctly add i18n to the component. Something like `Bênção?` turns into `Bencao` 🤣

Instead of changing the behaviour for these 2 `_.` cases, I think we should "solve" it for all cases. `startCase` should be removed all together, in favour of `textTransform: "capitalize"` which does the same capitalisation.

This way the user has control over which characters are filtered or not, and can even customise/disable the capitalisation by overriding the styles

Reply:
On my last push,
Removed the use of `startCase` function of lodash library and `removeExtension` function. 
Added css style: `textTransform: capitalize`.